drivers/segger/Kconfig

```diff
@@ -173,8 +173,15 @@ config SERIAL_RTT2
 		This option is used to enable the serial driver of channel 2
 
 config SERIAL_RTT_CONSOLE
-	int "Segger RTT console channel"
+	bool "Segger RTT console support"
```

Comment:
why not set SERIAL_RTT_CONSOLE to -1 for this case?

Reply:
This change is more for consistency than to fix anything.

This follows the scheme used by other console options where the option type is bool (SERIAL_CONSOLE, LWL_CONSOLE, RPMSG_UART_CONSOLE). 

With `SERIAL_RTT_CONSOLE=-1` we have to change
https://github.com/apache/nuttx/blob/7b18f5eb6f3af2dad5ef714e07b22181e22f3870/arch/arm/src/common/arm_internal.h#L57

to 
```
#  elif CONFIG_SERIAL_RTT_CONSOLE != -1
``` 
which differs from the rest of the conditions in this part of the code.

Reply:
Ok